src/components/Comments/Comment.js

```diff
@@ -318,7 +319,7 @@ const styles = StyleSheet.create({
   container: ({isLast, style, level, showLeftConnector}) => ({
     width: '100%',
     borderLeftWidth: showLeftConnector ? 1 : 0,
-    borderLeftColor: isLast ? (level === 0 ? COLORS.gray9 : 'transparent') : COLORS.gray9,
+    borderLeftColor: isLast ? (level === 0 ? colors.gray1 : 'transparent') : colors.gray1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The border color has been updated. Make sure this change doesn't affect the visibility and aesthetics of the border in different themes or backgrounds.
```diff
- borderLeftColor: isLast ? (level === 0 ? COLORS.gray9 : 'transparent') : COLORS.gray9,
+ borderLeftColor: isLast ? (level === 0 ? colors.gray1 : 'transparent') : colors.gray1,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->